gopath_copy.go

```diff
@@ -39,18 +38,6 @@ func CopyGopath(packageName, newGopath string, keepTests bool) error {
 		}
 	}
 
-	if !keepTests {
```

Comment:
I compiled a fairly large project with debugging output at this line and had not a single file removed by  `removeUnusedPkgs()`. Are you sure this is needed? Can you provide an example where this would remove files?

Reply:
Isn't the same check done at [Line 24](https://github.com/unixpickle/gobfuscate/blob/4fdc564a457cad4b9f45f35b3cc11aa54e173bf7/gopath_copy.go#L24)  and the [`containsDep()`](https://github.com/unixpickle/gobfuscate/blob/4fdc564a457cad4b9f45f35b3cc11aa54e173bf7/gopath_copy.go#L120) check is also implicitly done at [Line 29](https://github.com/unixpickle/gobfuscate/blob/4fdc564a457cad4b9f45f35b3cc11aa54e173bf7/gopath_copy.go#L29 )

Reply:
I think to copy the dependencies of tests in the first place this:
```go
if keepTests {
	srcFiles = append(srcFiles, pkg.TestGoFiles, pkg.XTestGoFiles)
}
```

Would need to become this:
```go 
if keepTests {
	srcFiles = append(srcFiles, pkg.TestGoFiles, pkg.XTestGoFiles, pkg.TestImports, pkg.XTestImports)
}
```

Reply:
Here is an example of a GOPATH that is affected by this change. Running `gobfuscate foo foo_out` gives this output on `master`:

```
2019/11/29 16:28:23 Copying GOPATH...
2019/11/29 16:28:24 Obfuscating package names...
2019/11/29 16:28:25 Obfuscating strings...
2019/11/29 16:28:25 Obfuscating symbols...
```

But this output if you remove the check at the bottom of `CopyGopath()`:

```
2019/11/29 16:28:23 Copying GOPATH...
2019/11/29 16:28:24 Obfuscating package names...
2019/11/29 16:28:25 Obfuscating strings...
2019/11/29 16:28:25 Obfuscating symbols...
Renamed 1 occurrence in 1 file in 1 package.
```

Notice how an unnecessary symbol is obfuscated without the check.

[test_gopath.tar.gz](https://github.com/unixpickle/gobfuscate/files/3906404/test_gopath.tar.gz)

Reply:
Ok thanks for clarification re added this check